pkgs/by-name/bi/binary/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  appstream,
+  desktop-file-utils,
+  glib,
+  gobject-introspection,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "binary";
+  version = "0.3";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "fizzyizzy05";
+    repo = "binary";
+    rev = version;
+    hash = "sha256-bR0oCqbnyUTCueT4f0Ij7qbwjNnN4eMDAOUK9MnCEJ0=";
+  };
+
+  nativeBuildInputs = [
+    appstream
+    desktop-file-utils
+    glib # need glib-compile-schemas
+    gobject-introspection
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [ libadwaita ];
+
+  dependencies = with python3Packages; [ pygobject3 ];
+
+  dontWrapGApps = true;
+  makeWrapperArgs = [ ''''${gappsWrapperArgs[@]}'' ];
```

double quotes are less tortured
```suggestion
  makeWrapperArgs = [ "\${gappsWrapperArgs[@]}" ];
```

---

this was killing me. thanks lol